src/main/java/bio/terra/cli/app/LocalProcessCommandRunner.java

```diff
@@ -56,7 +56,8 @@ protected String wrapCommandInSetupCleanup(List<String> command) {
    */
   protected int runToolCommandImpl(String command, Map<String, String> envVars)
       throws PassthroughException {
-    if (System.getProperty(CommandRunner.IS_TEST).equals("true")) {
+    String isTestSystemProperty = System.getProperty(CommandRunner.IS_TEST);
+    if (isTestSystemProperty != null && isTestSystemProperty.equals("true")) {
```

Comment:
can we just do TextUtils.isEqual(System.getProperty(CommandRunner.IS_TEST), "true")  or just
"true".equals(System.getProperty(CommandRunner.IS_TEST))

Reply:
Done